README.md

```diff
@@ -1,68 +1,69 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Hack Zero
+> Hackathon management web application <br>use in tandem with [Hack-Zero-Backend](https://github.com/hackthehack/hack-zero-v2-backend)
 
-## Available Scripts
+[![NPM Version][npm-image]][npm-url]
+[![Build Status][travis-image]][travis-url]
 
-In the project directory, you can run:
+Hack Zero is a serverless hackathon management system. It allows entities to create a single location to share
+and update participates on the details and happenings of a hackathon event.
+Participates are also able to sign up to gain access to a range of features.
 
-### `npm start`
+### Key Features
+ * User Registration
+ * User Login
+ * Hack Idea Creation
+ * Join Hack Idea Team
+ * Editing Hacks
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+![](header.png)
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+## Installation
 
-### `npm test`
+First download Node.js from this location [Here](https://nodejs.org/en/)
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+OS X & Linux & Windows:
 
-### `npm run build`
+```sh
+git clone https://github.com/hackthehack/hack-zero-v2
+```
+Once the app folder has finished downloading simply install the dependencies using the command bellow
+```sh
+npm install
+```
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+## Usage
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Hack Zero hack can be loaded into an S3 bucket for simple and quick deployment. An entity can deploy the application ahead of a hackathon event and request users to sign up. Once the user has signed up they can create ideas for the upcoming hackathon and create teams. The entity can then use contentful to push updates about the event to the users home page.
 
-### `npm run eject`
+## Development setup
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+You will need to install Node.js in order to run this application, please refer to the installation above
+<br>
+The .env file will need to be updated with the correct API url:
+```sh
+REACT_APP_API_URL=[url]
+```
+the Contentful connection will also beed to be updated in the .env file:
```

Comment:
Do they need to setup anything within their contentful space?

Reply:
I'll double check with Stanley